docs/writings/web-apps/run/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+---
+services:
+  traefik:
+    image: traefik:v2.9.1
```

Comment:
The example is outdated!!
Why not re-using the new one which is refactored and is on the latest stage?
You could create a `hello` example based on ocis_full which would make imho much more sense.
See: https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full

Reply:
thanks, i will take care

Reply:
Also traefik 2 is outdated, current is 3.1.2